lib/plugins/treatmentnotify.js

```diff
@@ -15,13 +15,13 @@ function init() {
     , pluginType: 'notification'
   };
 
-  //automated treatments from OpenAPS shouldn't trigger notifications or snooze alarms
+  //automated treatments from OpenAPS or Loop shouldn't trigger notifications or snooze alarms
   function filterTreatments (sbx) {
     var treatments = sbx.data.treatments;
 
     treatments = _.filter(treatments, function notOpenAPS (treatment) {
       var ok = true;
-      if (treatment.enteredBy && treatment.enteredBy.indexOf('openaps://') === 0) {
+      if (treatment.enteredBy && treatment.enteredBy.indexOf('openaps://' || 'loop://') === 0) {
```

Comment:
This is not correct @Kdisimone . You cannot use OR on a string. Suggested fix:

if (treatment.enteredBy && (treatment.enteredBy.indexOf('openaps://') === 0 || treatment.enteredBy.indexOf('loop://'))

Reply:
Thank you. I actually thought I’d closed this out because when I tested it the fix hadn’t worked. I’ll make the edit in the next couple hours and comment back. Thank you!!

Reply:
Or rather submit a new PR 😉